src/components/Button/FolderButton/FolderButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { MouseEvent, ReactNode } from "react";
+import styles from "./folderButton.module.css";
+
+interface Props {
+  children: ReactNode;
+  onClick: (e: MouseEvent<HTMLButtonElement>) => void;
+  ischecked: boolean;
+}
+
+function FolderButton({ children, onClick, ischecked }: Props) {
```

Comment:
함수 선언식으로 컴포넌트를 선언하는 경우에 export default를 파일의 아래에 두지 않고 요기에 같이 써도 괜찮은데 요건 어떠실까요~?
```suggestion
export default function FolderButton({ children, onClick, ischecked }: Props) {
```